src/modules/callouts/callouts.route.ts

```diff
@@ -21,4 +21,13 @@ export const calloutsRoute: Array<RouteRecordRaw> = [
       noAuth: true,
     },
   },
+  {
+    path: '/admin/callouts',
```

Comment:
We don't have anything else at `/admin`, correct?

Reply:
No but the create callout flow needs to move to `/admin/callouts/:id` and probably the notices list page to `/admin/notices`